src/components/Integrations/Shopify/Shopify.js

```diff
@@ -99,6 +81,31 @@ const Shopify = ({ intl, dependencies: { shopifyClient } }) => {
     </>
   );
 
+  const getData = useCallback(async () => {
+    const result = await shopifyClient.getShopifyData();
```

Comment:
I think that, before update the status, we need to determine if it has changed or not, in that way, we would avoid unnecessary and problematic rendering like this:

![2019-08-02_17-04-04 (1)](https://user-images.githubusercontent.com/1157864/62411280-f203d400-b5c6-11e9-8191-9c4c1b9dded0.gif)

Reply:
It is not happening anymore, we could check it again before the merge. Resolving the conversation...